lib/associations/belongs-to-many.js

```diff
@@ -287,6 +287,16 @@ class BelongsToMany extends Association {
        */
       removeMultiple: 'remove' + plural,
       /**
+       * Un-associate all instances.
+       *
+       * @param {Object} [options] Options passed to `through.destroy`
```

Comment:
The API should be like `removeMultiple`, e.g. it should accept an array of instances.
We could make it optional for `removeMultiple` as well and remove/destroy all associated if no array is passed.